README.md

```diff
@@ -598,6 +598,10 @@ git commit --fixup <SHA-1>
 ```sh
 git rebase -i --autosquash
 ```
+## skip staging during commit
+```sh
+git commit -a -m 'commit message'
```

Comment:
This can be `git commit -am 'commit message'` wonder how this basic tip was missed!